data_diff/__main__.py

```diff
@@ -99,8 +101,8 @@ def main(
         logging.error("Error while parsing age expression: %s" % e)
         return
 
-    table1 = TableSegment(db1, (table1_name,), key_column, update_column, columns, **options)
-    table2 = TableSegment(db2, (table2_name,), key_column, update_column, columns, **options)
+    table1 = TableSegment(db1, parse_table_name(table1_name), key_column, update_column, columns, **options)
+    table2 = TableSegment(db2, parse_table_name(table2_name), key_column, update_column, columns, **options)
```

Comment:
I think to your previous point, to make the Python API easier to use, what do you think about doing this inside of TableSegment?

Reply:
I think most users shouldn't create a TableSegment. They can instead call something like "create_table" which will do all the init and parsing necessary.

Reply:
OK, as long as we use that here too